test/unit/rng.test.js

```diff
@@ -12,7 +12,8 @@ describe('rng', () => {
     }
   });
 
-  test('Browser without crypto.getRandomValues()', () => {
+  // Newer versions of Node.js have a global crypto object
+  (global.crypto == null ? test : xtest)('Browser without crypto.getRandomValues()', function () {
```

Comment:
Thank you for looking into this!

I’m not quite sure though if this change is so useful. I think originally we tried to model the case where no global crypto object is present in which case we expect the library to throw in browser environments…

Is there any way to fake such an environment in the node test suite, e.g. by somehow unsetting the global crypto object?

Otherwise I‘m wondering if we should remove this test case altogether since we don‘t provide any polyfills for old browsers anyways and browsers without crypto (that was only IE<10) don’t play a role in practice anymore.